src/Components/SearchField/index.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState, useRef } from 'react';
+import { FiSearch } from 'react-icons/fi';
+import './styles.css';
+
+export function SearchField(props) {
+
+    const inputRef = useRef(null);
+    const handleSearch = () => {
+        alert(inputRef.current.value);
+    }
+
+    return (
+        <div className="search-button-container">
+            <input type="text" placeholder={props.buttonTitle} ref={ inputRef }/>
+            <button onClick={handleSearch}>
+                <FiSearch size="1.5em" />
```

Comment:
```suggestion
                <FiSearch size="1.5rem" />
```
O _em_ ele compara o tamanho da fonte com o tamanho da fonte da div que ele está presente, é melhor utilizar rem, o rem ele pega o tamanho da fonte root do projeto